src/sdk/resources/policyItemActivities.ts

```diff
@@ -0,0 +1,9 @@
+import { IHttpClient } from '../http';
+import { PagedResource } from './resource';
+import { PolicyItemActivity } from '../models/policies';
+
+export class PolicyItemActivitiesResource extends PagedResource<PolicyItemActivity> {
```

Comment:
There will be no activities endpoint in phase 1. That all still needs to be designed and fleshed out.

Reply:
I was using this to set the text on the `Activities` column on the items grid. Otherwise, I'd have the activity id on the item, right? Should I remove this column from the items grid (for phase 1)?

Reply:
Yes